### PR TITLE
Formatting fixes

### PR DESCRIPTION
**Pull request by LtSnail**

### CODE REVIEW

File: CoreLib/tests.cpp [hunk 1]
```diff
@@ -50,32 +50,27 @@ TEST(CoreLibTests, UniquePtr_ReleaseTest)
...
+        void operator()(int* ptr) { delete ptr; }
     };
     UniquePtr<int, CustomDeleter> Ptr(new int(5));
     EXPECT_TRUE(Ptr);
 }
 
 TEST(CoreLibTests, UniquePtr_ArrayTest)
 {
-    UniquePtr<int[]> Ptr(new int[3] {1, 2, 3});
+    UniquePtr<int[]> Ptr(new int[3]{1, 2, 3});
```
Comment (Mon1Amour4) [inline CoreLib/tests.cpp:63]: It looked better

File: Engine/Fuego/Application.cpp [hunk 2]
```diff
@@ -28,31 +28,24 @@ Application::Application() : d(new ApplicationImpl())
     d->m_Running = true;
 
     // Temp::
-    float vertices[3 * 3] = {
-        -0.5f, -0.5f, 0.0f, 0.5f, -0.5f, 0.0f, 0.0f, 0.5f, 0.0f};
+    float vertices[3 * 3] = {-0.5f, -0.5f, 0.0f, 0.5f, -0.5f, 0.0f, 0.0f, 0.5f, 0.0f};
```
Comment (Mon1Amour4) [inline Engine/Fuego/Application.cpp:31]: For Objects like this we have to turn off clang-format -> format how we like it and then turn it on
{ 1, 1, 1,
  0, 0, 0,
}
I dont think so we can automate it but we can agree on style for such cases

File: Engine/Fuego/Application.cpp [hunk 3]
```diff
@@ -28,31 +28,24 @@ Application::Application() : d(new ApplicationImpl())
...
+void Application::PushOverlay(Layer* overlay) { d->m_LayerStack.PushOverlay(overlay); }
 
 void Application::OnEvent(Event& event)
 {
     EventDispatcher dispatcher(event);
-    dispatcher.Dispatch<WindowCloseEvent>(
-        std::bind(&Application::OnWindowClose, this, std::placeholders::_1));
-    dispatcher.Dispatch<WindowResizeEvent>(
-        std::bind(&Application::OnWindowResize, this, std::placeholders::_1));
+    dispatcher.Dispatch<WindowCloseEvent>(std::bind(&Application::OnWindowClose, this, std::placeholders::_1));
```
Comment (Mon1Amour4) [inline Engine/Fuego/Application.cpp:47]: May we change to smth like this?:
dispatcher.Dispatch<WindowCloseEvent>(std::bind(&Application::OnWindowClose, this,
                                                                    std::placeholders::_1));

File: Engine/Fuego/Input.h [hunk 4]
```diff
@@ -8,24 +8,15 @@ namespace Fuego
...
-        return m_Instance->IsKeyPressedImpl(keyCode);
-    };
+public:
+    static inline bool IsKeyPressed(KeyCode keyCode) { return m_Instance->IsKeyPressedImpl(keyCode); };
 
-    static inline bool IsMouseButtonPressed(MouseCode mouseCode)
-    {
-        return m_Instance->IsMouseButtonPressedImpl(mouseCode);
-    };
+    static inline bool IsMouseButtonPressed(MouseCode mouseCode) { return m_Instance->IsMouseButtonPressedImpl(mouseCode); };
```
Comment (LepusAlleni) [inline Engine/Fuego/Input.h:14]: Easier to read
```
static inline bool IsMouseButtonPressed(MouseCode mouseCode) 
{ 
    return m_Instance->IsMouseButtonPressedImpl(mouseCode); 
}
```

`;` should not be after function definition

File: Engine/Fuego/Windows/OpenGL/OpenGLContext.cpp [hunk 5]
```diff
@@ -2,8 +2,7 @@
 
 namespace Fuego
 {
-OpenGLContext::OpenGLContext(HWND* windowHandle)
-    : _windowHandle(windowHandle), _openGLContext(nullptr)
+OpenGLContext::OpenGLContext(HWND* windowHandle) : _windowHandle(windowHandle), _openGLContext(nullptr)
```
Comment (LepusAlleni) [inline Engine/Fuego/Windows/OpenGL/OpenGLContext.cpp:5]: One element per line, optionally with comma before elements

File: Engine/Fuego/Windows/WindowWin.cpp [hunk 6]
```diff
@@ -50,63 +49,53 @@ LRESULT WindowWin::WindowProc(HWND hwnd, UINT msg, WPARAM wparam, LPARAM lparam)
         }
 
         case WM_LBUTTONDOWN:
```
Comment (LepusAlleni) [inline Engine/Fuego/Windows/WindowWin.cpp:51]: Remove `case` extra padding (switch on the case level)

**Comment by LepusAlleni** [comment]
`MaxEmptyLinesToKeep: 2` or `3`

**Comment by LepusAlleni** [comment]
Since clang-format 16:
`InsertNewlineAtEOF: true`

**Review by LepusAlleni — CHANGES_REQUESTED** [review]

File: Engine/Fuego/Windows/WindowWin.cpp (hunk 6 above)
Comment (LepusAlleni) [reply Engine/Fuego/Windows/WindowWin.cpp:51]: `IndentCaseLabels: false`

File: Engine/Fuego/Application.cpp (hunk 3 above)
Comment (LtSnail) [reply Engine/Fuego/Application.cpp:48]: Sorry, didn't get this one

**Review by LepusAlleni — APPROVED** [review]

**Pull request merged**